ios/rudder_sdk_flutter.podspec

```diff
@@ -12,7 +12,7 @@ RudderStack flutter SDK ios plugin project
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency "Rudder", ">= 1.1.2"
+  s.dependency "Rudder", ">= 1.2.1"
```

Comment:
We have decided, we will be keep minimum version 1.0 for Rudder SDK. Can you make it `~> 1.0`?

Reply:
The current minimum version in master is 1.2.1, do you mean you'd like it to be even lower?

It was upgraded to `1.2.1`. 18 days ago and before it was `1.1.2`. 2 months ago it was `1.0.24`

Reply:
Yes. That's correct.